resources/views/dashboard.twig

```diff
@@ -55,6 +55,7 @@ function deleteTalk(tid) {
         <div>
             <a class="btn btn-white-o mr-2" href="{{ url('password_edit') }}">Change Password</a>
             <a class="btn btn-white-o mr-2" href="{{ url('user_edit', { id: user.id }) }}">Edit Profile</a>
+            <a class="btn btn-white-o mr-2" href="{{ url('user_delete') }}">Delete Profile</a>
```

Comment:
When I freshly signed up, the dashboard shows the following:

![screen shot 2018-05-22 at 08 36 15](https://user-images.githubusercontent.com/605483/40345875-34c6298a-5d9b-11e8-9d32-6be0446ab00d.png)

Also see https://github.com/opencfp/opencfp/blob/2d3ea6a061ac31ae0c73090d02a3e4edae73177f/resources/views/dashboard.twig#L25-L32

That is, unless newly signed-up users fill out their personal details, they cannot see the link tot the *Delete Profile* page. What do you think about adjusting `dashboard.twig` to also show the link when the profile is not filled out yet?